scripts/process-model-data.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+"""Process CSV model data, e.g. for aggregation or merging
+
+Usage:
+    scripts/process-model-data.py [-h] <query-name> <csv-dir>
+
+Options:
+    -h --help       Show this screen.
+"""
+import sys
+sys.path.insert(0, '.')
+
+from docopt import docopt
+import pandas as pd
+import os.path
+
+from dmscripts.models import queries
+
+
+CONFIG = {
+    'successful_supplier_applications': queries.successful_supplier_applications,
+    'briefs_with_summary_application_data': queries.briefs_with_summary_application_data
+}
+
+
+def get_query(query_name):
+    query_func = CONFIG.get(query_name)
+    if query_func is None:
+        raise ValueError('{} is not a defined query.'.format(query_name))
+    return query_func
+
+if __name__ == '__main__':
+    arguments = docopt(__doc__)
+
+    CSV_DIR = arguments['<csv-dir>']
+    QUERY_NAME = arguments['<query-name>']
+
+    query = get_query(QUERY_NAME)
+    data_frames = dict()
+    # query params give us the table (data frame) names we need...
+    for data_frame_name in query.__code__.co_varnames[:query.__code__.co_argcount]:
```

Comment:
This feels backwards to me, but it's probably coming back to the same point I've made above.

I don't think function is a good format to *store* configuration, which is what this is essentially using it for. I think we need to generalise the logic and explicitly provide model/file names as parameters instead of introspecting them from argument names.